_layouts/email.html

```diff
@@ -6,8 +6,12 @@ <h2>{{ page.name }}</h2>
 		<b>{{ page.city }}, {{ page.state }}</b><br /> {{ site.email_intro_text | markdownify }}
 		<p id="autoopen">{{ site.auto_open_message }}</p>
 		<div class='buttons'>
-			<a href="javascript:void(0);" onclick="openEmail()">Send email</a>
-			<a href="javascript:void(0);" onclick="copyToClipboard(this, `{{ page.permalink }}`, true)">Copy link</a>
+      <a href="javascript:void(0);" onclick="openEmail('mailto')">Send email</a>
+      <a href="javascript:void(0);" onclick="copyToClipboard(this, `{{ page.permalink }}`, true)">Copy link</a></br>
+			Send via webmail (
+        <a href="javascript:void(0);" onclick="openEmail('gmail')">gmail</a>
+        <a href="javascript:void(0);" onclick="openEmail('yahoo')">yahoo mail</a>
+      )
```

Comment:
There seems to be a space before gmail and after yahoo mail (which should also be capitalized)